indico/legacy/pdfinterface/latex_templates/inc/contribution.tex

```diff
@@ -180,16 +180,18 @@
         \setlength{\leftskip}{0pt}
     \JINJA{endif}
 
-    \JINJA{for field_value in contrib.field_values}
-        \textbf{\VAR{ field_value.contribution_field.title}}:
+    \JINJA{for field in contrib.event.contribution_fields if field.is_active and field.is_public}
```

Comment:
I think this change results in fields that had no value (e.g. because the field was created after the contribution) showing up